sdk/python/jobs/pipelines/1h_automl_in_pipeline/automl-forecasting-in-pipeline/automl-forecasting-in-pipeline.ipynb

```diff
@@ -201,19 +453,388 @@
+    "# Download run's artifacts/outputs\n",
+    "local_path = mlflow_client.download_artifacts(\n",
+    "    mlflow_best_run.info.run_id, \"outputs\", local_dir\n",
+    ")\n",
+    "print(\"Artifacts downloaded in: {}\".format(local_path))\n",
+    "print(\"Artifacts: {}\".format(os.listdir(local_path)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Get metrics for each runs\n",
+    "In this code we list all runs, sharing the same parent run ID and ending by the underscore followed by the order number."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import re\n",
+    "\n",
+    "parent_run_id = ml_model[\"run_id\"][: ml_model[\"run_id\"].index(\"_\")]\n",
+    "child_run_regex = re.compile(r\"[^_]+_\\d+$\")\n",
+    "\n",
+    "for child_run in filter(\n",
+    "    lambda x: child_run_regex.match(x.name),\n",
+    "    ml_client.jobs.list(parent_job_name=parent_run_id),\n",
+    "):\n",
+    "    print(\n",
+    "        f\"{child_run.name}: \"\n",
+    "        f'{mlflow_client.get_run(child_run.name).data.metrics[\"normalized_root_mean_squared_error\"]}'\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3. Inference"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are several ways to do the inference, for here we will demonstrate how to use the registered model and pipeline to do the inference. (how to register a model https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.model.model?view=azure-ml-py)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.1 Get Inference Pipeline Environment\n",
+    "To trigger an inference pipeline run, we first need a running environment for run that contains all the appropriate packages for the model unpickling. This environment can be created using the `yaml`, which we have downloaded with other than best run artifacts and which is located at ```artifact_downloads``` directory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import Environment\n",
+    "\n",
+    "env = Environment(\n",
+    "    description=\"environment for automl inference\",\n",
+    "    image=\"mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20210727.v1\",\n",
+    "    conda_file=os.path.join(\"artifact_downloads\", \"outputs\", \"conda_env_v_1_0_0.yml\"),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.2 Build and submit the inference pipeline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The inference pipeline will create pipline output object, which can be downloaded after pipeline finishes."
```

Comment:
- nit: pipeline
- please remove the comma after the word object

Reply:
Fixed.